### PR TITLE
elixir: update to 1.15.7

### PR DESCRIPTION
**Pull request by pguyot**
###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 14.0 23A344 x86_64
Xcode 15.0 15A240d

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint --nitpick`?
- [x] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?
- [ ] checked that the Portfile's most important [variants](https://trac.macports.org/wiki/Variants) haven't been broken?

### CODE REVIEW

File: lang/elixir/Portfile [hunk 1]
```diff
@@ -3,7 +3,7 @@
 PortSystem          1.0
 PortGroup           github 1.0
 
-github.setup        elixir-lang elixir 1.15.6 v
+github.setup        elixir-lang elixir 1.15.7 v
 github.tarball_from archive
 epoch               1
```
Comment (herbygillot) [inline lang/elixir/Portfile:8]: Can you add a `revision` field?

```suggestion
revision            0
epoch               1
```

**Comment by herbygillot** [comment]
Also can you make the homepage URL HTTPS?

File: lang/elixir/Portfile (hunk 1 above)
Comment (pguyot) [reply lang/elixir/Portfile:9]: Sure. The current preferred format is to always include revision? At some point we wanted to skip it when it was 0.

Comment (herbygillot) [reply lang/elixir/Portfile:9]: I think I've recently seen that we want to have it explicit and present, even if set to zero.

**Comment by herbygillot** [comment]
Thanks @pguyot

**Pull request merged**